src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main (String[] args) {
```

Comment:
Please remove the space between the method name 'main' and the parentheses to follow Java naming conventions.